DTS-STN/Alpha-Site
# Analytics load issue

# Description

[#380 Make sure page loads are being tracked after the first page load for analytics](https://trello.com/c/Z8tmWpF0/380-380-make-sure-page-loads-are-being-tracked-after-the-first-page-load-for-analytics)

There was two page loads being called that messed up the analytics tracking, I forgot to remove the one in _app.js so that was the issue.

## Acceptance Criteria

Check to see if _app.js doesn't add a page load event

## Test Instructions

1. Run the code and go on the splash page
2. Click a language to go to the home page
3. Open your console and write adobeDataLayer (The array should have 2 items and not 3)

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
